results/sentence-transformers__all-MiniLM-L6-v2/model.json

```diff
@@ -0,0 +1 @@
+{"model_name": "sentence-transformers/all-MiniLM-L6-v2", "time_of_run": "2024-03-18 11:22:22.739054", "versions": {"sentence_transformers": "2.0.0", "transformers": "4.6.1", "pytorch": "1.8.1"}}
```

Comment:
Maybe add a revision number to make sure it's the same model version that is used? Wdyt?

Reply:
I was thinking about doing the same as for datasets in mteb (`revision_id`). Just specifying  the commit id from the HF repo that stores the model.

Reply:
Yes I would love to do that. However, I am not sure the commit id is available in the model object (you would have to know it beforehand). I would love to add that (but seems like that is outside the scope of this PR)

@imenelydiaker do anyone from your team have the time to give it a go?

Reply:
I've seen this in the `transformers` doc: https://huggingface.co/docs/transformers/main_classes/model#transformers.PreTrainedModel.from_pretrained.revision
But it's for the `PreTrainedModel.from_pretrained()` function. I can check how it can be applied for `SentenceTransformers` 🙂